common/containers/Tabs/WalletDecrypt/index.jsx

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import translate from 'translations';
+import KeystoreDecrypt from './KeystoreDecrypt';
+import PrivateKeyDecrypt from './PrivateKeyDecrypt';
+import MnemonicDecrypt from './MnemonicDecrypt';
+import LedgerNanoSDecrypt from './LedgerNanoSDecrypt';
+import TrezorDecrypt from './TrezorDecrypt';
+import ViewOnlyDecrypt from './ViewOnlyDecrypt';
+
+const decryptionChoices = [
```

Comment:
Maybe we should move this to a separate config file entirely? Kinda nitpicky but I think it might be nice to limit react components to only the JS logic, and limit the amount of config data.